in-a-box.py

```diff
@@ -155,7 +160,7 @@ def select_scm_provider():
         'scm_plugin': scm_plugin
     }
 
-def generate_oauth(scm_plugin, ip):
+def generate_scm_config(scm_plugin, ip):
```

Comment:
I commented [here](https://github.com/screwdriver-cd/screwdriver/pull/648/files#r131034209) as well, the default settings in default.yaml may not be effective if the setting is a array format. So I think that `in-a-box.py` must generate the whole settings for `scms`, and rename the function.